src/components/TextPopup/TextPopup.js

```diff
@@ -65,11 +68,21 @@ const TextPopup = () => {
     >
       <CustomizablePopup dataElement="textPopup">
         <ActionButton
-          dataElement="copyTextButton"
-          title="action.copy"
-          img="ic_copy_black_24px"
-          onClick={copyText}
+          dataElement="editTextButton"
+          title="action.edit"
+          img="ic_edit_black_24px"
+          onClick={() =>
+            dispatch(actions.openElement('editTextModal'))
+          }
         />
+        {fullAPI && (
```

Comment:
I think we always want to show the copy text button, but only show the edit text button when fullAPI is enabled.